httpserver/http_connection.cpp

```diff
@@ -962,14 +973,15 @@ void connection_t::process_client_req() {
     const std::string client_ip =
         socket_.remote_endpoint().address().to_string();
     if (rate_limiter_.should_rate_limit_client(client_ip)) {
+        this->body_stream_ << "too many requests\n";
         response_.result(http::status::too_many_requests);
-        LOKI_LOG(error, "Rate limiting client request.");
+        LOKI_LOG(debug, "Rate limiting client request.");
         return;
     }
 
 #ifndef DISABLE_ENCRYPTION
     if (!parse_header(LOKI_EPHEMKEY_HEADER)) {
-        LOKI_LOG(error, "Could not parse headers");
+        LOKI_LOG(debug, "Could not parse headers");
```

Comment:
Here and below could we make all these messages consistent by starting with `Bad client request: ...`

Reply:
updated